cluster/gce/util.sh

```diff
@@ -1878,7 +1884,8 @@ function generate-konnectivity-server-certs {
 # $1 master env (kube-env of master; result of calling get-master-env)
 # $2 env key to use
 function get-env-val() {
-  local match=`(echo "${1}" | grep -E "^${2}:") || echo ""`
+  local match
+  match=`(echo "${1}" | grep -E "^${2}:") || echo ""`
```

Comment:
Please do not use legacy backticks, but use contemporary $() instead.

https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03